gateway/cli/src/general_commands.rs

```diff
@@ -93,6 +94,20 @@ pub enum GeneralCommands {
     },
     /// Safely stop the gateway
     Stop,
+    /// List the transactions that the gateway has processed
+    Transactions {
```

Comment:
Should the command not be named Payments similar to the event names? Or even PaymentLog to be more descriptive?

Reply:
`PaymentLog` is good, I can change it to that